data/i18n/en/en.toml

```diff
@@ -151,6 +151,12 @@ other = "Until"
 [feature_state]
 other = "FEATURE STATE:"
 
+[feature_state_kubernetes_label]
+other = "Kubernetes"
+
+[feature_state_feature_gate_tooltip]
+other = "Feature Gate:"
```

Comment:
These don't localize as well as they could do. Something we could improve after a merge IMO.